src/js/components/CreateAccountForm.js

```diff
@@ -145,61 +157,71 @@ class CreateAccountForm extends Component {
       this.state.email
         ? false
         : true;
+
     return (
-      <Form className="CreateAccount-form">
-        <div className="CreateAccount-nameFields">
+      <div className="CreateAccount-form">
+        <Form id="CreateAccountForm">
+          <div className="CreateAccount-nameFields">
+            <FormFieldHelper
+              baseClass="CreateAccount"
+              idPrefix="create-account"
+              name="first name"
+              onChange={this.handleFirstNameChange}
+            />
+            <FormFieldHelper
+              baseClass="CreateAccount"
+              idPrefix="create-account"
+              name="last name"
+              onChange={this.handleLastNameChange}
+            />
+          </div>
           <FormFieldHelper
             baseClass="CreateAccount"
             idPrefix="create-account"
-            name="first name"
-            onChange={this.handleFirstNameChange}
+            name="email"
+            onChange={this.handleEmailChange}
           />
+          {errors.emailFormat.length > 0 ? (
+            <Message negative content={errors.emailFormat} />
+          ) : null}
           <FormFieldHelper
             baseClass="CreateAccount"
             idPrefix="create-account"
-            name="last name"
-            onChange={this.handleLastNameChange}
+            name="password"
+            type="password"
+            onChange={this.handlePasswordChange}
+          />
+          <FormFieldHelper
+            baseClass="CreateAccount"
+            idPrefix="create-account"
+            name="confirm password"
+            type="password"
+            onChange={this.handlePasswordConfirmChange}
+          />
+          {errors.passwordMatch.length > 0 ? (
+            <Message negative content={errors.passwordMatch} />
+          ) : null}
+          <Button
+            positive
+            className="CreateAccount-submit"
+            type="submit"
+            onClick={this.handleCreateAccount}
+            disabled={enableSubmit}
+          >
+            Create Account
+          </Button>
+          {errors.emailTaken.length > 0 ? (
+            <Message negative content={errors.emailTaken} />
+          ) : null}
+        </Form>
+        {this.state.registrationComplete ? (
+          <Message
+            success
+            header={"Your user registration was successful"}
+            content={<a href="/">You may now login!</a>}
```

Comment:
nice! idk what `<a>` tags look like within a semantic-ui Message but you could add a class if it needs any styling.